website/pages/docs/auth/jwt_oidc_providers.mdx

```diff
@@ -128,6 +128,79 @@ Main reference: [Using OAuth 2.0 to Access Google APIs](https://developers.googl
 1. Configure Authorized Redirect URIs.
 1. Save client ID and secret.
 
+### Google-specific handling configuration
+
+Provider specific configuration is available when using Google as an identity provider from the
+Vault JWT/OIDC auth method. The configuration allows Vault to obtain G Suite group membership and
+user information during the JWT/OIDC authentication flow. The group membership obtained from G Suite
+may be used for Identity group alias association. The user information obtained from G Suite can be
+used to copy claims data into resulting auth token and alias metadata via [claim_mappings](/api/auth/jwt#claim_mappings).
+
+#### Setup
+
+To set up the Google-specific handling, you'll need:
+- A G Suite account with the [super admin role](https://support.google.com/a/answer/2405986?hl=en)
+for granting domain-wide delegation API client access.
+- The ability to create a service account in [Google Cloud Platform](https://console.developers.google.com/iam-admin/serviceaccounts).
+
+The Google-specific handling that's used to fetch G Suite groups and user information in Vault uses
+[G Suite Domain-Wide Delegation of Authority](https://developers.google.com/admin-sdk/directory/v1/guides/delegation)
+for authentication and authorization. You need to follow **all steps** in the [guide](https://developers.google.com/admin-sdk/directory/v1/guides/delegation)
+to obtain a Google service account key file capable of making requests to the G Suite
+[User Accounts](https://developers.google.com/admin-sdk/directory/v1/guides/manage-users) and
+[Groups](https://developers.google.com/admin-sdk/directory/v1/guides/manage-groups) APIs.
+
+In **step 5** within the section titled
+[Delegate domain-wide authority to your service account](https://developers.google.com/admin-sdk/directory/v1/guides/delegation#delegate_domain-wide_authority_to_your_service_account),
+the only OAuth scopes that should be granted are:
+- `https://www.googleapis.com/auth/admin.directory.group.readonly`
+- `https://www.googleapis.com/auth/admin.directory.user.readonly`
+
+This is an **important security step** in order to give the service account the least set of privileges
+that enable the feature.
```

Comment:
I wonder if it'd be worth calling this out a little more by using the `~>` syntax, like we've done here: 

https://github.com/hashicorp/vault/blob/4c44ccd52ab5bb7bb43d4a16235a18bf7e9f9634/website/pages/docs/platform/k8s/helm/configuration.mdx#L124-L125

Just a thought, I think it's also good the way it is.

Reply:
Good idea. I like that better. Done in d6b6f6909e41e1862b8718b3431c4e3bbb3df722.